proto/config/storage.proto

```diff
@@ -208,6 +208,10 @@ message Volume {
   string displayName = 7;  // Optional friendly name echo'ed in info message
   bool clear_text = 8;     // Flag to indicate the volume encryption needed or not
   Target target = 9;       // How the volume will be presented to the instance. At the moment only the AppCustom is used
+
+  // To inform the edge-node if the device receiving this Volume is
+  // responsible for volume creation, convert PVC, or not
+  string designated_node_id = 10;
```

Comment:
See comment about this field for the app instance.

Reply:
i put the reply in above app-instance comment.

Reply:
FWIW I still think this will cause technical debt since the controller has to be involved in re-selecting a designated node if the previously selected one fails. So approving this holding my nose.